web/packages/teleport/src/TrustedClusters/TrustedClusters.tsx

```diff
@@ -63,14 +71,16 @@ export default function TrustedClusters() {
       <FeatureHeader alignItems="center">
         <FeatureHeaderTitle>Trusted Clusters</FeatureHeaderTitle>
         {hasClusters && (
-          <ButtonPrimary
```

Comment:
There's another place where `ButtonPrimary` is used in this file. Should it be removed as well?

Reply:
No, because this is an empty state with an actual CTA (the user has no trusted clusters, therefore this is the action that we recommend).